src/webots/nodes/utils/WbObjectDetection.hpp

```diff
@@ -49,8 +50,8 @@ class WbObjectDetection {
   bool isContainedInFrustum(const WbAffinePlane *frustumPlanes);
 
   // Computes the frustum plane for the given device ray.
-  static WbAffinePlane *computeFrustumPlanes(const WbSolid *device, const double verticalFieldOfView,
-                                             const double horizontalFieldOfView, const double maxRange);
+  static WbAffinePlane *computeFrustumPlanes(const WbSolid *device, double verticalFieldOfView, double horizontalFieldOfView,
+                                             double maxRange, bool isPlanarProjection);
```

Comment:
`const` keywords were removed in the master PR. I believe these can be restored to improve the code readability.